lib/msf/core/auxiliary/mqtt.rb

```diff
@@ -0,0 +1,68 @@
+# -*- coding: binary -*-
+
+require 'msf/core/exploit'
+require 'rex/proto/mqtt'
+
+module Msf
+  module Auxiliary::MQTT
+      def initialize(info = {})
+        super
+
+        register_options(
+          [
+            Opt::RPORT(Rex::Proto::MQTT::DEFAULT_PORT),
+            OptString.new('USERNAME', [false, 'The user to authenticate as']),
+            OptString.new('PASSWORD', [false, 'The password to authenticate with'])
+          ]
+        )
+
+        register_advanced_options(
+          [
+            OptString.new('CLIENT_ID', [false, 'The client ID to send if necessary for bypassing clientid_prefixes']),
+            OptInt.new('READ_TIMEOUT', [true, 'Seconds to wait while reading MQTT responses', 5])
+          ]
+        )
+
+        register_autofilter_ports([Rex::Proto::MQTT::DEFAULT_PORT, Rex::Proto::MQTT::DEFAULT_PORT])
```

Comment:
Should this be `[Rex::Proto::MQTT::DEFAULT_PORT, Rex::Proto::MQTT::DEFAULT_SSL_PORT]`?